test/powershell/Modules/Microsoft.PowerShell.Utility/Tee-Object.Tests.ps1

```diff
@@ -18,7 +27,16 @@ Describe "Tee-Object" -Tags "CI" {
 	        Write-Output teeobjecttest3  | Tee-Object $teefile
 	        Get-Content $teefile | Should -BeExactly "teeobjecttest3"
 	        Remove-Item $teefile -ErrorAction SilentlyContinue
-	    }
+        }
+
+        It "Parameter 'Encoding' should accept encoding" {
+            $encoding = 1251
+            $teefile = $testfile
+            $testvalue | Tee-Object -Encoding $encoding $teefile
+            Get-Content $teefile -Encoding $encoding | Should -BeExactly $testvalue
+            (Get-Content $teefile -AsByteStream) -join "-" | Should -BeExactly $expectedBytes
+            Remove-Item -Path $teefile -ErrorAction SilentlyContinue -Force
```

Comment:
Since we use TestDrive (previous comment) and we need remove the test file before each test please move this to `BeforeEach` Pester block for all 3 `It` blocks.